docroot/profiles/ilr/ilr.install

```diff
@@ -502,3 +502,12 @@ function ilr_update_7042() {
     'ilr_forms_students',
   ));
 }
+
+/**
+ * Enables ilr_forms_public feature
+ */
+function ilr_update_7043() {
```

Comment:
7043 now exists on master. Please update to 7044, pull and rebase master and push again. Thanks.